manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
@niteshkumarniranjan Great work! Just one more thing, add icons to the manifest file.

Reply:
@akshatnitd No We don't have to clear the cache but instead we have to change the cache name.